From cd7615c760acbc6cf81b0cb1bbc9136c8dd7e4d3 Mon Sep 17 00:00:00 2001
From: Jurgen Kramer <kramer@orwell.fritz.box>
Date: Sat, 18 Feb 2017 10:42:05 +0100
Subject: [PATCH] Add native DSD support for HDTA Serenade DSD

Add native DSD support for the HDTA Serenade DSD DAC.

---
 sound/usb/quirks.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/usb/quirks.c b/sound/usb/quirks.c
index 01eff6c..4c39372 100644
--- a/sound/usb/quirks.c
+++ b/sound/usb/quirks.c
@@ -1356,6 +1356,7 @@ u64 snd_usb_interface_dsd_format_quirks(struct snd_usb_audio *chip,
 	case USB_ID(0x20b1, 0x2009): /* DIYINHK DSD DXD 384kHz USB to I2S/DSD */
 	case USB_ID(0x20b1, 0x2023): /* JLsounds I2SoverUSB */
 	case USB_ID(0x20b1, 0x3023): /* Aune X1S 32BIT/384 DSD DAC */
+	case USB_ID(0x22e1, 0xxa01): /* HDTA Serenade DSD */
 	case USB_ID(0x2616, 0x0106): /* PS Audio NuWave DAC */
 		if (fp->altsetting == 3)
 			return SNDRV_PCM_FMTBIT_DSD_U32_BE;
-- 
2.9.3

